Modules/Finance/SubModules/AccountingBatches.cs

```diff
@@ -21,5 +21,15 @@ public CWRequest ExportRequest(int batchId, string serializedExportAccountingBat
         {
             return new CWRequest(CWHttpMethod.Post, $"{getPrefix()}/{batchId}/export", serializedExportAccountingBatchRequest);
         }
-    }
+
+		/// <summary>
+		/// Export the payload data from an existing batch.
+		/// </summary>
+		/// <param name="exportAccountingBatchParameters">ExportAccountingBatchParameters object to be included in the body of the request.</param>
+		/// <returns>CWRequest to be sent using CWHttpClient.</returns>
+		public CWRequest ExportRequest(object exportAccountingBatchParameters)
```

Comment:
I would prefer if this endpoint mimicked the one above and took in the already serialized `string` of the body content rather than receiving an `object`. 

Also the CW REST API documentation isn't super helpful in this regard but we might need to be more informative in the `<summary>` of this endpoint to clarify how it differs from the above endpoint.. I don't really use CWM API daily anymore so I'm not understanding how they differ just looking at the docs. But maybe this is a non-issue.

Also going to nitpick the extra whitespace here.

Reply:
It's a very important difference. Exporting should be done through a 2-phase step on CW.